ww4ff/model/metric.py

```diff
@@ -27,3 +27,15 @@ def mcc(self) -> float:
         if denom == 0:
             denom = 1
         return num / denom
+
+    @property
+    def fpr(self) -> float:
```

Comment:
For FAR (FPR), we want it computed as number of false positives per hour of (nonoverlapping) input speech.

Reply:
so there are two cases 1) frame level, 2) wake-word level
for frame level, I guess total speech length would be something like the number of sample * stride size
for wake-word level, it is going to be the total length of the speech.

Should this function support both cases? if so what might be the best way to handle them both?